sdk/core/logger/package.json

```
@@ -34,6 +34,7 @@
   },
   "files": [
     "dist/",
+    "!dist/**/*.d.*ts.map",
```

I am curious, why is the last `*` needed?
```suggestion
    "!dist/**/*.d.ts.map",
```

---

we have some `*-browser.mts`.  I could probably use `!dist/**/*.d.[c,m]*ts.map`